src/shared/lib/svgToPdfDownload.ts

```diff
@@ -1,5 +1,6 @@
 import svg2pdf from "svg2pdf.js";
 import {jsPDF} from "jspdf-yworks";
+import font from "font.json";
```

Comment:
lets change the name of this file to something more descriptive.  like pdf-font.   or perhaps the name of the actual font.

Reply:
Got it, changed the name.